nixos/modules/services/misc/transfer-sh.nix

```
@@ -0,0 +1,422 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.transfer-sh;
+in
+{
+  options = {
+    services.transfer-sh = {
+      enable = mkEnableOption "transfer-sh setup";
+
+      package = mkPackageOption pkgs "transfer-sh" { };
+
+      environmentFile = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = lib.mdDoc "Environment file as defined in {manpage}`systemd.exec(5)`.";
```

```suggestion
        description = "Environment file as defined in {manpage}`systemd.exec(5)`.";
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557